src/components/inputs/Combobox/Combobox.tsx

```diff
@@ -144,6 +146,7 @@ const Combobox = (props: IComboboxProps) => {
 		onChange,
 		options = {},
 		optionHeight = 's',
+		maxHeightOffest = 0,
```

Comment:
Should this be `...Offset` instead of `...Offest`?

Reply:
Yep! And as to why this is necessary - we dynamically set the maxheight for the dropdown based on the Window height (so that it doesn't overflow the window). It stops like 75 px above the window bottom. But in the case of Cloud Backups, we don't really want to cover up the stepper, because it looks bad. So this allows us to decrease the max height.